packages/e2e/test/wallet/PersonalWallet/phase2validation.test.ts

```diff
@@ -59,7 +59,10 @@ describe('PersonalWallet/phase2validation', () => {
     wallet.shutdown();
   });
 
-  it('can detect a phase2 validation failure and emit the transaction as failed', async () => {
+  // The submit fails instead of being able to send it.
+  // TODO: re-enable this after understanding how to force submit API to submit a transaction that
```

Comment:
Do we have to use cardano-submit-api in e2e? Maybe we should hoist it under a separate configuration-specific group of tests (like the long-running group that we have). Either way, I suggest creating a ticket to resolve this.